core/src/test/kotlin/org/evomaster/core/output/TestCaseWriterTest.kt

```diff
@@ -0,0 +1,289 @@
+
+        val expectedLines = Lines()
+        expectedLines.add("@Test")
+        expectedLines.add("public void test() throws Exception {")
+        expectedLines.indent()
+        expectedLines.add("List<InsertionDto> insertions = sql().insertInto(\"myTable\").d(\"Column0\", \"stringValue0\").d(\"Column1\", \"stringValue1\").dtos();")
+        expectedLines.add("controller.execInsertionsIntoDatabase(insertions);")
+        expectedLines.deindent()
+        expectedLines.add("}")
+
+        assertEquals(expectedLines.toString(), lines.toString())
+    }
+
+
+    @Test
+    fun testIntegerColumn() {
+        val idColumn = Column("Id", "INTEGER", 10, primaryKey = false)
+        val nameColumn = Column("Name", "VARCHAR", 10, primaryKey = false)
+
+        val aTable = Table("myTable", setOf(idColumn, nameColumn), HashSet<ForeignKey>())
+
+        val id = 0L
+
+        val integerGene = IntegerGene(idColumn.name, 42, 0, 10)
+        val stringGene = StringGene(nameColumn.name, "nameValue", 0, 10)
+
+        val insertIntoTableAction = DbAction(aTable, setOf(idColumn, nameColumn), id, listOf(integerGene, stringGene))
+
+        val (format, baseUrlOfSut, ei) = buildEvaluatedIndividual(mutableListOf(insertIntoTableAction))
+
+        val test = TestCase(test = ei, name = "test")
+
+        val writer = TestCaseWriter()
+
+        val lines = writer.convertToCompilableTestCode(format, test, baseUrlOfSut)
+
+        val expectedLines = Lines()
+        expectedLines.add("@Test")
+        expectedLines.add("public void test() throws Exception {")
+        expectedLines.indent()
+        expectedLines.add("List<InsertionDto> insertions = sql().insertInto(\"myTable\").d(\"Id\", 42).d(\"Name\", \"nameValue\").dtos();")
+        expectedLines.add("controller.execInsertionsIntoDatabase(insertions);")
+        expectedLines.deindent()
+        expectedLines.add("}")
+
+        assertEquals(expectedLines.toString(), lines.toString())
+    }
+
+
+    @Test
+    fun testPrimaryKeyColumn() {
+        val idColumn = Column("Id", "INTEGER", 10, primaryKey = true)
+        val nameColumn = Column("Name", "VARCHAR", 10, primaryKey = false)
+
+        val aTable = Table("myTable", setOf(idColumn, nameColumn), HashSet<ForeignKey>())
+
+
+        val id = 0L
+
+        val integerGene = IntegerGene(idColumn.name, 42, 0, 10)
+        val primaryKeyGene = SqlPrimaryKeyGene(idColumn.name, "myTable", integerGene, 10)
+        val stringGene = StringGene(nameColumn.name, "nameValue", 0, 10)
+
+        val insertIntoTableAction = DbAction(aTable, setOf(idColumn, nameColumn), id, listOf(primaryKeyGene, stringGene))
+
+        val (format, baseUrlOfSut, ei) = buildEvaluatedIndividual(mutableListOf(insertIntoTableAction))
+
+        val test = TestCase(test = ei, name = "test")
+
+        val writer = TestCaseWriter()
+
+        val lines = writer.convertToCompilableTestCode(format, test, baseUrlOfSut)
+
+        val expectedLines = Lines()
```

Comment:
this can be simplified with ".apply{}", so you do not have to repeat expectedLines